azure-pipelines.yml

```diff
@@ -228,6 +229,45 @@ jobs:
       TargetPath: 'build_scripts/windows/out/'
       ArtifactName: msi-$(Platform)
 
+- job: BuildWindowsZIP
+  displayName: Build Windows ZIP
+  strategy:
+    matrix:
+      x64:
+        Platform: x64
```

Comment:
I agree we should only release 64-bit ZIP as 32-bit is on its way to deprecation.